### DISCUSSION TITLE
Allow seeing starred repos that are not in any list, or not in some lists

### DISCUSSION THREAD

**Discussion by LouisCAD** [community/community#8840] [Other Feature Feedback, Questions, & Ideas]
This would be very helpful to sort our stars.

**Comment by Morphclue** [comment]
An automatic `unlisted` category could be a good solution.

**Comment by CBenoit** [comment]


**Reply by Morphclue** [reply]
Hey @CBenoit avoid comments that do not add any new information and are only used for approval.
You can approve with suggestions by upvoting a solution. This reduces the Mails that are being send out to every user watching this thread.

**Comment by ivadham** [comment]
Alternative title: Unlisted stars as a list.

**Comment by movahhedi** [comment]
Still waiting for this feature...

**Comment by ksharizard** [comment]
Hi! I was facing the same issue with having a lot of stars and not knowing which are unlisted. I created a bash script to get a user's stars and lists and find the unlisted stars. I tested this on my profile and it worked well to output the unlisted repos. Just linking it here if anybody finds it interesting and wants to try it out:

https://github.com/ksharizard/github-lists
